app/assets/v2/css/base.css

```diff
@@ -1126,20 +1126,17 @@ input.is-invalid {
 }
 
 #alpha{
-    color: #a78600;
+    color: #000000;
```

Comment:
Bookkeeping but think we are just using `color: black;' throughout the codebase. @the-lostonemc ?

Reply:
Well I believe we had agreed to stick to the hex color, so as to keep it uniform throughout ! Else we end up with names in a few areas and hex code in the rest !

Reply:
Whoops! @willsputra please disregard :)